### ISSUE TITLE
Expecto: run `Illegal argument: line must be positive`

### ISSUE THREAD

**Issue by haf** [ionide/ionide-vscode-fsharp#346]
<img width="1010" alt="screen shot 2017-01-25 at 10 25 49" src="https://cloud.githubusercontent.com/assets/193115/22286891/f479f102-e2f0-11e6-8611-7127691bcc3e.png">

At https://github.com/logary/logary/commit/5378a9d8cd73c99ff8feb474a405406d0af36edb if you `cd src/tests/Logary.Tests && code .` and then run `Expecto run` and then `Expecto watch` in there.

**Comment by Krzysztof-Cieslak** [comment]
Checking out this commit and trying to build projects end up with:
```
"d:\Programowanie\Projekty\logary\src\tests\Logary.Tests\Logary.Tests.fsproj" (default target) (1) ->
"d:\Programowanie\Projekty\logary\src\Logary\Logary.fsproj" (default target) (2) ->
(CoreCompile target) -> 
  d:\Programowanie\Projekty\logary\src\Logary\Internals\Service.fs(32,1): error FS0037: Duplicate definition of type, exception or module 'Service' [d:\Programowanie\Projekty\logary\src\Logary\Logary.fsproj]
  d:\Programowanie\Projekty\logary\src\Logary\Registry.fs(48,13): error FS0039: The field, constructor or member 'create' is not defined [d:\Programowanie\Projekty\logary\src\Logary\Logary.fsproj]
  d:\Programowanie\Projekty\logary\src\Logary\Registry.fs(248,50): error FS0039: The type 'T' is not defined [d:\Programowanie\Projekty\logary\src\Logary\Logary.fsproj]
  d:\Programowanie\Projekty\logary\src\Logary\Registry.fs(248,82): error FS0039: The type 'T' is not defined [d:\Programowanie\Projekty\logary\src\Logary\Logary.fsproj]
  d:\Programowanie\Projekty\logary\src\Logary\Registry.fs(253,48): error FS0039: The field, constructor or member 'getState' is not defined [d:\Programowanie\Projekty\logary\src\Logary\Logary.fsproj]
  d:\Programowanie\Projekty\logary\src\Logary\Registry.fs(293,45): error FS0039: The type 'T' is not defined [d:\Programowanie\Projekty\logary\src\Logary\Logary.fsproj]
  d:\Programowanie\Projekty\logary\src\Logary\Registry.fs(293,45): error FS0039: The type 'T' is not defined [d:\Programowanie\Projekty\logary\src\Logary\Logary.fsproj]
```

Any idea?

**Comment by haf** [comment]
I see. Thanks, I'll work a bit more on that project, make sure CI is passing and the notify you again in this thread.

**Comment by Krzysztof-Cieslak** [comment]
Fixed in `2.23.5`
